src/routes/landing/components/FilterableLocationSelector.css

```diff
@@ -1,27 +1,10 @@
-@import url('../../../modules/common/constants/colors.css');
-
 :root {
   --hover-color: rgba(0, 0, 0, 0.1);
   --item-height: 30px;
   --list-margin: 30px;
 }
 
-.language-list-item {
-  display: block;
-  padding: 7px;
-  color: inherit;
-  text-decoration: inherit;
-
-  &:hover {
-    color: inherit;
-    text-decoration: inherit;
-    transition: background-color 0.5s ease;
-    background-color: var(--hover-color);
-  }
+.top-spacing {
```

Comment:
I think this should still be part of LandingPage. This would make the component more reusable.

Reply:
I agreed with @maxammann that this is ok